package-lock.json

```diff
@@ -1,8 +1,7811 @@
 {
```

Comment:
@mpiniarski could you please remove the `package-lock` file from the PR, it shouldn't be committed as it's unrelated